### ISSUE TITLE
Lag + console error for "unknown biomes", need workaround. 

### ISSUE THREAD

**Issue by purejosh** [GriefPrevention/GriefPrevention#1773]
### Observed Behavior

Going through nether portals is extremely laggy, and you receive a console error stating that the biome you're landing in isn't a known biome. 

### Expected Behavior

Resolve lag, or give us a way to disable whatever feature is causing this bug, for server owners who use custom biomes. 

### Reproduction steps

1. install terralith worldgen and GP
2. go through portal to land in a biome that's custom
3. experience lag and console error

### Stack trace or error log

```shell
[16:06:10 WARN]: [GriefPrevention] Plugin GriefPrevention v16.17.1-2b69191 generated an exception while executing task 565280
java.lang.NoSuchFieldError: DESERT_HILLS
	at me.ryanhamshire.GriefPrevention.RestoreNatureProcessingTask.getPlayerBlocks(RestoreNatureProcessingTask.java:835) ~[GriefPrevention.jar:?]
	at me.ryanhamshire.GriefPrevention.AutoExtendClaimTask.lambda$getBiomeBlocks$0(AutoExtendClaimTask.java:78) ~[GriefPrevention.jar:?]
	at java.util.HashMap.computeIfAbsent(HashMap.java:1220) ~[?:?]
	at me.ryanhamshire.GriefPrevention.AutoExtendClaimTask.getBiomeBlocks(AutoExtendClaimTask.java:78) ~[GriefPrevention.jar:?]
	at me.ryanhamshire.GriefPrevention.AutoExtendClaimTask.getLowestBuiltY(AutoExtendClaimTask.java:57) ~[GriefPrevention.jar:?]
	at me.ryanhamshire.GriefPrevention.AutoExtendClaimTask.run(AutoExtendClaimTask.java:32) ~[GriefPrevention.jar:?]
	at org.bukkit.craftbukkit.v1_18_R1.scheduler.CraftTask.run(CraftTask.java:101) ~[paper-1.18.1.jar:git-Paper-175]
	at org.bukkit.craftbukkit.v1_18_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57) ~[paper-1.18.1.jar:git-Paper-175]
	at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[paper-1.18.1.jar:git-Paper-175]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

### Server version

```shell
[16:19:23 INFO]: This server is running Paper version git-Paper-175 (MC: 1.18.1) (Implementing API version 1.18.1-R0.1-SNAPSHOT) (Git: 9490475)
You are running the latest version
Previous version: git-Paper-76 (MC: 1.18.1)
```

### GriefPrevention version

```shell
[16:19:56 INFO]: GriefPrevention version 16.17.1-2b69191
```

### Configuration

```yaml
# Default values are perfect for most servers.  If you want to customize and have a question, look for the answer here first: http://dev.bukkit.org/bukkit-plugins/grief-prevention/pages/setup-and-configuration/

GriefPrevention:
  SeaLevelOverrides:
    world: 260
    world_nether: 260
    world_the_end: 260
    seventeen: -1
    oldnether: -1
    creative: -1
  Claims:
    Mode:
      world_the_end: Survival
      creative: Creative
      world_nether: Survival
      world: Survival
      oldnether: Disabled
      seventeen: Survival
    PreventGlobalMonsterEggs: true
    PreventTheft: true
    ProtectCreatures: true
    PreventButtonsSwitches: true
    LockWoodenDoors: false
    LockTrapDoors: false
    LockFenceGates: true
    EnderPearlsRequireAccessTrust: true
    RaidTriggersRequireBuildTrust: true
    ProtectHorses: true
    ProtectDonkeys: true
    ProtectLlamas: true
    InitialBlocks: 100
    Claim Blocks Accrued Per Hour:
      Default: 100
    Max Accrued Claim Blocks:
      Default: 80000
    Accrued Idle Threshold: 0
    AccruedIdlePercent: 0
    AbandonReturnRatio: 1.0
    AutomaticNewPlayerClaimsRadius: 4
    AutomaticNewPlayerClaimsRadiusMinimum: 0
    ExtendIntoGroundDistance: 5
    MinimumWidth: 5
    MinimumArea: 100
    MaximumDepth: 0
    InvestigationTool: GOLDEN_AXE
    ModificationTool: GOLDEN_SHOVEL
    Expiration:
      ChestClaimDays: 30
      UnusedClaimDays: 30
      AllClaims:
        DaysInactive: 60
        ExceptWhenOwnerHasTotalClaimBlocks: 10000
        ExceptWhenOwnerHasBonusClaimBlocks: 5000
      AutomaticNatureRestoration:
        SurvivalWorlds: false
    AllowTrappedInAdminClaims: false
    MaximumNumberOfClaimsPerPlayer: 0
    CreationRequiresWorldGuardBuildPermission: true
    VillagerTradingRequiresPermission: true
    CommandsRequiringAccessTrust: /sethome
    DeliverManuals: true
    ManualDeliveryDelaySeconds: 30
    RavagersBreakBlocks: true
    FireSpreadsInClaims: false
    FireDamagesInClaims: false
    LecternReadingRequiresAccessTrust: true
  Spam:
    Enabled: false
    LoginCooldownSeconds: 60
    LoginLogoutNotificationsPerMinute: 5
    ChatSlashCommands: /me;/global;/local
    WhisperSlashCommands: /tell;/pm;/r;/whisper;/msg
    WarningMessage: Please reduce your noise level.  Spammers will be banned.
    BanOffenders: true
    BanMessage: Banned for spam.
    AllowedIpAddresses: 1.2.3.4; 5.6.7.8
    DeathMessageCooldownSeconds: 60
    Logout Message Delay In Seconds: 0
  PvP:
    RulesEnabledInWorld:
      world: false
      world_nether: false
      world_the_end: false
      seventeen: true
      oldnether: true
      creative: true
    ProtectFreshSpawns: true
    PunishLogout: true
    CombatTimeoutSeconds: 15
    AllowCombatItemDrop: false
    BlockedSlashCommands: /home;/vanish;/spawn;/tpa
    ProtectPlayersInLandClaims:
      PlayerOwnedClaims: true
      AdministrativeClaims: true
      AdministrativeSubdivisions: true
    AllowLavaDumpingNearOtherPlayers:
      PvPWorlds: true
      NonPvPWorlds: false
    AllowFlintAndSteelNearOtherPlayers:
      PvPWorlds: true
      NonPvPWorlds: false
    ProtectPetsOutsideLandClaims: false
  Economy:
    ClaimBlocksMaxBonus: 0
    ClaimBlocksPurchaseCost: 0.0
    ClaimBlocksSellValue: 0.0
  ProtectItemsDroppedOnDeath:
    PvPWorlds: false
    NonPvPWorlds: false
  BlockLandClaimExplosions: false
  BlockSurfaceCreeperExplosions: true
  BlockSurfaceOtherExplosions: false
  LimitSkyTrees: false
  LimitTreeGrowth: false
  PistonMovement: EVERYWHERE
  PistonExplosionSound: false
  FireSpreads: true
  FireDestroys: true
  AdminsGetWhispers: true
  AdminsGetSignNotifications: true
  SmartBan: true
  Mute New Players Using Banned Words: true
  MaxPlayersPerIpAddress: 3
  Siege:
    Worlds: []
    BreakableBlocks:
    - GRASS_BLOCK
    - DIRT
    - COBBLESTONE
    - OAK_PLANKS
    - SPRUCE_PLANKS
    - BIRCH_PLANKS
    - JUNGLE_PLANKS
    - ACACIA_PLANKS
    - DARK_OAK_PLANKS
    - SAND
    - GRAVEL
    - GLASS
    - GRASS
    - FERN
    - DEAD_BUSH
    - WHITE_WOOL
    - ORANGE_WOOL
    - MAGENTA_WOOL
    - LIGHT_BLUE_WOOL
    - YELLOW_WOOL
    - LIME_WOOL
    - PINK_WOOL
    - GRAY_WOOL
    - LIGHT_GRAY_WOOL
    - CYAN_WOOL
    - PURPLE_WOOL
    - BLUE_WOOL
    - BROWN_WOOL
    - GREEN_WOOL
    - RED_WOOL
    - BLACK_WOOL
    - SNOW
    - GLASS_PANE
    DoorsOpenDelayInSeconds: 300
    CooldownEndInMinutes: 60
  EndermenMoveBlocks: true
  SilverfishBreakBlocks: true
  CreaturesTrampleCrops: true
  RabbitsEatCrops: true
  HardModeZombiesBreakDoors: false
  Database:
    URL: ''
    UserName: ''
    Password: ''
  UseBanCommand: false
  BanCommandPattern: ban %name% %reason%
  Advanced:
    fixNegativeClaimblockAmounts: true
    ClaimExpirationCheckRate: 60
    OfflinePlayer_cache_days: 90
  Abridged Logs:
    Days To Keep: 7
    Included Entry Types:
      Social Activity: true
      Suspicious Activity: true
      Administrative Activity: false
      Debug: false
      Muted Chat Messages: false
```

### Plugin list

```shell
[16:20:49 INFO]: Plugins (27): AngelChest, BetterLogStrip, BetterTridents, ChestCommands, Chunky, CombatWeaponryPlus, CoreProtect, DoorsReloaded, Economy, EconomyShopGUI, GravityControl, GriefPrevention, ItemsAdder, JukeBoxPlus, LightAPI, LoneLibs, LPC, LuckPerms, Multiverse-Core, Multiverse-Inventories, Multiverse-SignPortals, PlayerShopGUIPlus, ProtocolLib, StackResize, Vault, voicechat, WorldEdit
```

### Running without GriefPrevention

- [ ] I attempted running the server without GriefPrevention installed.
- [ ] The problem does _not_ occur when GriefPrevention is removed from the server.

### Running with only GriefPrevention

- [ ] I attempted running only GriefPrevention on the server.
- [ ] The issue still occurs when GriefPrevention is the only plugin running.

### Running on a fresh, clean server installation

- [ ] I attempted testing for the issue on a new server.
- [ ] The issue still occurs on a new server.

### Using unmodified client

- [ ] I attempted testing for the issue with the vanilla client.
- [ ] The issue still occurs when using the vanilla client.

### We appreciate you taking the time to fill out a bug report!

- [X] I searched for similar issues before submitting this bug report.

**Comment by Jikoo** [comment]
Duplicate of #1689

**Comment by purejosh** [comment]
This is not the same error. Why did you close this? 

Also, #1689 is unsolved and closed.

**Comment by Jikoo** [comment]
It's the same error, and was resolved by #1694.
